packages/web3-core-helpers/package.json

```diff
@@ -9,16 +9,29 @@
     },
     "types": "types/index.d.ts",
     "scripts": {
-        "dtslint": "dtslint types --onlyTestTsNext"
+        "dtslint": "dtslint types --onlyTestTsNext",
+        "build:dev": "../../scripts/build/build.dev.sh",
+        "build:cjs": "../../scripts/build/build.cjs.sh",
+        "build:esm": "../../scripts/build/build.esm.sh",
+        "build:minified": "../../scripts/build/build.min.sh",
+        "watch": "../../scripts/build/watch.dev.sh"
     },
-    "main": "src/index.js",
+    "main": "dist/web3-core-helpers.cjs.js",
+    "module": "dist/web3-core-helpers.esm.js",
+    "unpkg": "dist/web3-core-helpers.min.js",
+    "jsdelivr": "dist/web3-core-helpers.min.js",
     "dependencies": {
+        "@babel/runtime": "^7.6.3",
         "underscore": "1.9.1",
         "web3-eth-iban": "1.2.2",
         "web3-utils": "1.2.2"
     },
     "devDependencies": {
+        "bn.js": "^4.11.8",
```

Comment:
Those ``devDependendencies`` are used to create small minified bundles. In other words to ``dedupe`` for example ``bn.js``.

Reply:
Can you elaborate? I don't follow.

What happens if I install `web3-core-helpers` in an empty npm project. How can that work?

Reply:
Because ``web3-core-helpers`` does have a dependency on ``web3-utils`` which does have dependencies with ``bn.js`` as a dependency and those "second level" ``bn.js`` dependencies will get duplicated if I don't specify custom resolver rules. This is the reason for those ``devDependencies`` and the ``dedupe`` option.

Reply:
I don't get it. But I don't have the complete picture of the rollup config in mind yet.